repos/testUtils/parkin/hooks.js

```diff
@@ -0,0 +1,18 @@
+const { parkin } = require('./instance')
```

Comment:
With all this configuration, it might make sense to move it into it's own repo, something like `parkin-jest`, similar to how `cucumber-jest` works
 
A lot of this code could be reused, if we ever used parkin in a different app

Reply:
Yeah I agree. We could really make it a jest transformer, like cucumber-jest.